bin/generate-tiles

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# For backward compatibility, allow both PG* and POSTGRES_* forms,
+# with the non-standard POSTGRES_* form taking precedence.
+# An error will be raised if neither form is given, except for the PGPORT
+export PGDATABASE="${POSTGRES_DB:-${PGDATABASE?}}"
+export PGUSER="${POSTGRES_USER:-${PGUSER?}}"
+export PGPASSWORD="${POSTGRES_PASSWORD:-${PGPASSWORD?}}"
+export PGPORT="${POSTGRES_PORT:-${PGPORT:-5432}}"
+
+# List of postgres servers
+# "xxx.xxx.xxx.xxx&host=xxx.xxx.xxx.xxx&host=..."
+if [[ -z "${PGHOSTS_LIST}" ]]
+then
+  export HOST_COUNT=1
+  export PGHOSTS="${POSTGRES_HOST:-${PGHOST?}}"
+else
+  export HOST_COUNT=`awk -F"&" '{print NF}' <<< "${PGHOSTS_LIST}"`
+  export PGHOSTS="${PGHOSTS_LIST}"
+fi
+
+export NO_GZIP=${NO_GZIP:-1}
+if [[ ${NO_GZIP} ]] && [[ ! ${NO_GZIP} == "0" ]] && [[ ! ${NO_GZIP} == "true" ]]
+then
+  export NO_GZIP=1
+fi
+
+export USE_KEY_COLUMN=${USE_KEY_COLUMN:-1}
+if [[ ${USE_KEY_COLUMN} ]] && [[ ! ${USE_KEY_COLUMN} == "0" ]] && [[ ! ${USE_KEY_COLUMN} == "true" ]]
+then
+  export USE_KEY_COLUMN=1
+fi
```

Comment:
Assuming the default for `USE_KEY_COLUMN` should be `USE_KEY_COLUMN=1`

```suggestion
case "${USE_KEY_COLUMN:-1}" in
  0 | false ) export USE_KEY_COLUMN= ;;
  * ) export USE_KEY_COLUMN=1 ;;
esac
```

If the default should be  `USE_KEY_COLUMN=`, use `case "${USE_KEY_COLUMN:-0}" in`